packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -408,15 +413,22 @@ export class MgtTodo extends MgtTasksBase {
       return;
     }
 
+    this._isLoadingTasks = true;
     if (!this._graph) {
       const graph = provider.graph.forComponent(this);
       this._graph = graph;
     }
 
-    const currentList = this.currentList;
-    if (currentList) {
-      await this.loadTasks(currentList);
+    if (this.targetId) {
```

Comment:
@sebastienlevert what should the default load state be? i.e. when neither target-id nor initial-id are set. Should we force the user to make a selection, or should we select the list with the wellknownListName of defaultList?

I'm personally divided in that if neither property is set then it makes sense for the picker to have nothing selected, but this is contrary to the behavior of the ToDo service where the default list is selected by default.

Reply:
@gavinbarron I'm also divided on this. I would prefer having a default selected as the default behavior. This would mean allowing not setting a default though? (These requests come in sooner or later 😆)

Reply:
Given that we have #2374 to track the default loading state here, we can complete this PR and handle the selection separately